Automattic/wp-calypso
# Tabbed navigation for wp.com plans and signup

This PR updates plans page both on /plans and in signup to have personal/business tabs.

Related to the blogger plan project p9jf6J-Sn-p2

**Testing instructions:**

1. Follow instructions in D20180-code and make sure you are in the `test` group
1. Confirm signup steps with plan selection looks like this:

<img width="1118" alt="zrzut ekranu 2018-10-29 o 15 58 08" src="https://user-images.githubusercontent.com/205419/47658563-7c7c5a80-db93-11e8-8d1e-18d46152c5bb.png">
<img width="949" alt="zrzut ekranu 2018-10-29 o 15 58 12" src="https://user-images.githubusercontent.com/205419/47658564-7d14f100-db93-11e8-9b2e-d0c9ca135d58.png">

1. Confirm plans page looks like this:

<img width="1077" alt="zrzut ekranu 2018-10-29 o 15 58 02" src="https://user-images.githubusercontent.com/205419/47658562-7c7c5a80-db93-11e8-980c-cbc02caa9db5.png">
<img width="1066" alt="zrzut ekranu 2018-10-29 o 15 57 24" src="https://user-images.githubusercontent.com/205419/47658560-7c7c5a80-db93-11e8-98c6-362f6390c672.png">

1. Follow instructions in D20180-code and make sure you are in the `control` group
1. Confirm that you can't any see tabs on `/plans` or during the signup
1. Confirm that jetpack signup flow works just like in production
